src/app/navbar/navbar.component.html

```diff
@@ -8,6 +8,45 @@
 
     <div class="navbar-collapse collapse">
       <app-search-bar></app-search-bar>
+      <form class="navbar-form navbar-left">
+        <div class="input-group" id="nav-group">
+          <input type="text" name="query" class="form-control" id="nav-input" placeholder="Search" (keyup.enter)="submit()" [(ngModel)]="searchdata.query">
```

Comment:
this is rendering an extra search bar remove these lines these are already included in app-search component 
![image](https://cloud.githubusercontent.com/assets/15216503/22968118/2ff9ca32-f38f-11e6-9313-22bc99110354.png)